Agents/FenlandTrajectoryAgent/README.md

```diff
@@ -0,0 +1,149 @@
+[OntoDevice]: https://www.theworldavatar.com/kg/ontodevice
+[TheWorldAvatar]: https://github.com/cambridge-cares/TheWorldAvatar
+[TimeSeriesClient]: https://github.com/cambridge-cares/TheWorldAvatar/tree/main/JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/timeseries
+[Stack Manager]: https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/stack-manager
+[Stack Manager README]: https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-manager/README.md
+[JPS_Document]: https://github.com/cambridge-cares/TheWorldAvatar/blob/dev-pydantic-rdflib/JPS_BASE_LIB/python_wrapper/docs/examples/additional_java_lib.md
+[twa]: https://pypi.org/project/twa/
+[virtual environment]: https://docs.python.org/3/tutorial/venv.html
+[Python wrapper]: https://github.com/cambridge-cares/TheWorldAvatar/tree/main/JPS_BASE_LIB/python_wrapper#installing-additional-java-resources
+[allows you to publish and install packages]: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-to-github-packages
+[CMCL Docker registry wiki page]: https://github.com/cambridge-cares/TheWorldAvatar/wiki/Docker%3A-Image-registry
+[Common stack scripts]: https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/common-scripts
+[Create SSH key]: https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/create-with-openssh/
+[DataPoint]: https://www.metoffice.gov.uk/services/data/datapoint/about
+[Github container registry]: https://ghcr.io
+[Github package repository]: https://github.com/cambridge-cares/TheWorldAvatar/wiki/Packages
+[http://localhost:5000/]: http://localhost:5000/
+[Java Development Kit version >=11]: https://adoptium.net/en-GB/temurin/releases/?version=11
+[JDBC driver]: https://jdbc.postgresql.org/download/ 
+[JPS_BASE_LIB]: https://github.com/cambridge-cares/TheWorldAvatar/tree/main/JPS_BASE_LIB
+[personal access token]: https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token
+[py4jps]: https://pypi.org/project/py4jps/#description
+[Stack Manager]: https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/stack-manager
+[spin up the stack]: https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-manager/README.md
+[Stack-Clients]: https://github.com/cambridge-cares/TheWorldAvatar/tree/dev-MetOfficeAgent-withinStack/Deploy/stacks/dynamic/stack-clients
+[TheWorldAvatar]: https://github.com/cambridge-cares/TheWorldAvatar
+[Upload SSH key]: https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/to-existing-droplet/
+[virtual environment]: https://docs.python.org/3/tutorial/venv.html
+[VSCode via SSH]: https://code.visualstudio.com/docs/remote/ssh
+
+<!-- files -->
+[Dockerfile]: ./Dockerfile
+[docker compose file]: ./docker-compose.yml
+[gps_target_folder]: ./agent/raw_data/gps_target_folder
```

Comment:
Thank you for the thorough review. I did consider option 2) first, and modifying it would affect both Flask app and the instantiation code. Please allow me to implement a temporary measure using option 1) in commit [0fc61f295e84eaf02e11c29b349077ca237f5c5a], so the URL no longer returns a 404. I am still debugging the changes required in option 2).